status-im/status-react
# [#4010] Allow both release and develop version to coexist

fixes #4010

### Summary:
Add ability to install debug and release build together.

### Steps to test:
- Install release build
- Install debug bulid
- Check that two apps exist: Status and Status Debug.

Wait for the new icon.

status: ready
